sagemaker-data-wrangler/schedule_Data_Wrangler_flow_using_Lambda_function/lambda_function.py

```diff
@@ -0,0 +1,183 @@
+            data_distribution_type="FullyReplicated",
+            # You can override below to point to other database or use different queries
+            athena_dataset_definition=AthenaDatasetDefinition(
+                catalog="AwsDataCatalog",
+                database="cms",
+                query_string=runtime_query_string,
+                output_s3_uri=f"s3://{source_bucket}/athena/Inpatient_Claim/",
+                output_format="PARQUET"
+            )
+        )
+    ))
+
+    data_sources.append(ProcessingInput(
+        source=f"s3://{source_bucket}/DW/DE1_0_2008_Beneficiary_Summary_File_Sample_20.csv",
+        # You can override this to point to other dataset on S3
+        destination="/opt/ml/processing/2008_Beneficiary_Summary",
+        input_name="2008_Beneficiary_Summary",
+        s3_data_type="S3Prefix",
+        s3_input_mode="File",
+        s3_data_distribution_type="FullyReplicated"
+    ))
+
+    # Sagemaker session
+    sess = sagemaker.Session()
+    print(f"Data Wrangler sagemaker session: {sess}")
+
+    # You can configure this with your own bucket name, e.g.
+    # bucket = <my-own-storage-bucket>
+    bucket = sess.default_bucket()
+    print(f"Data Wrangler export storage bucket: {bucket}")
+
+    # unique flow export ID
+    flow_export_id = f"{time.strftime('%Y-%m-%d-%H-%M-%S', time.gmtime())}-{str(uuid.uuid4())[:8]}"
+    flow_export_name = f"flow-{flow_export_id}"
+
+    # Output name is auto-generated from the select node's ID + output name from the flow file.
+    output_name = "8b392709-d2c4-4b8e-bdda-e75b2d14f35e.default"
+
+    s3_output_prefix = f"export-{flow_export_name}/output"
+    s3_output_path = f"s3://{bucket}/{s3_output_prefix}"
+    print(f"Flow S3 export result path: {s3_output_path}")
+
+    s3_processing_output = ProcessingOutput(
+        output_name=output_name,
+        source="/opt/ml/processing/output",
+        destination=s3_output_path,
+        s3_upload_mode="EndOfJob"
+    )
+
+    # name of the flow file which should exist in the current notebook working directory
+    flow_file_name = "cms.flow"
+
+    with open(flow_file_name) as f:
+        flow = json.load(f)
+
+    # Upload flow to S3
+    s3_client = boto3.client("s3")
+    s3_client.upload_file(flow_file_name, bucket, f"data_wrangler_flows/{flow_export_name}.flow")
+
+    flow_s3_uri = f"s3://{bucket}/data_wrangler_flows/{flow_export_name}.flow"
+
+    print(f"Data Wrangler flow {flow_file_name} uploaded to {flow_s3_uri}")
+
+    # Input - Flow: cms.flow
+    flow_input = ProcessingInput(
+        source=flow_s3_uri,
+        destination="/opt/ml/processing/flow",
+        input_name="flow",
+        s3_data_type="S3Prefix",
+        s3_input_mode="File",
+        s3_data_distribution_type="FullyReplicated"
+    )
+
+    print(f"ProcessingInput defined")
+
+    # IAM role for executing the processing job. You can set this up with your own role
+    iam_role = "<IAM-ROLE-PLACEHOLDER>"
```

Comment:
Kind of buried and a required edit by the customer.... can this be tied to sagemaker session instead and assume they've setup their roles... and then this script can be tested?